update_version.sh

```diff
@@ -117,7 +117,7 @@ updateLatestVersion() {
 # thank you https://www.vinaygopinath.me/blog/tech/commit-to-master-branch-on-github-using-travis-ci/
 git clone "https://github.com/${VERSIONS_REPOSITORY}.git"
 cd "${REPOSITORY_NAME}" || { echo "'${REPOSITORY_NAME}' dir not found"; exit 1; }
-git config user.email "$( echo "${GITHUB_USERNAME}" | awk '{print tolower($0)}' )-ci@not-real.com"
+git config user.email "${GITHUB_USERNAME,,}-ci@not-real.com"
```

Comment:
please revert to awk

Reply:
Done, I also caught this in `check_tag.sh` and `update_insider`.